### PR TITLE
Tabs-Components

### PR DESCRIPTION
**Pull request by ogwurujohnson**

### CODE REVIEW

File: components/Tabs/Tabs.js [hunk 1]
```diff
@@ -59,4 +68,7 @@ class TabItem {
 
 */
 
-links = document.querySelectorAll();
\ No newline at end of file
+links = document.querySelectorAll('.tabs-link');
+links.forEach(link => {
+  return new TabLink(link);
```
Comment (mattshardman) [inline components/Tabs/Tabs.js]: you can one line arrow functions

```links.forEach(link => new TabLink(link))```

**Comment by mattshardman** [comment]
Great job Johnson!

- You have correctly implemented the drop down elements and the tabs
- Your git work flow is amazing

**Possible things to work on**
- Not much to say, you have correctly implemented all of the MVP features. Just make sure you understand the underlying theory and could create this from scratch.
- Melvine included a tab for you but you didn't give him one. Bit mean lol

Overall great job, you have implemented everything correctly. . Keep up the good work!

File: components/Tabs/Tabs.js (hunk 1 above)
Comment (ogwurujohnson) [reply components/Tabs/Tabs.js]: fixed, Thanks

**Comment by ogwurujohnson** [comment]
Lol, thanks. Would keep practicing to get a good hang of it

**Pull request merged**